addons/pos_restaurant/static/src/overrides/components/order_widget/order_widget.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module */
+
+import { OrderWidget } from "@point_of_sale/app/generic_components/order_widget/order_widget";
+import { patch } from "@web/core/utils/patch";
+import { _t } from "@web/core/l10n/translation";
+
+patch(OrderWidget.props, {
+    ...OrderWidget.props,
+    isConfigRestaurant: { type: Boolean, optional: true },
+    isOrderBooked: { type: Boolean, optional: true },
+});
```

Comment:
```suggestion
patch(OrderWidget, {
    props: {
        ...OrderWidget.props,
        isConfigRestaurant: { type: Boolean, optional: true },
        isOrderBooked: { type: Boolean, optional: true },
    }
});
```
Sorry about my previous comment. This is now the correct patch to the static fields of the components.